ui/app/AppLayouts/Chat/ChatColumn/ChatMessages.qml

```diff
@@ -83,12 +83,13 @@ ScrollView {
             }
         }
         MouseArea {
-           cursorShape: Qt.PointingHandCursor
-           anchors.fill: newMessagesBox
-           onClicked: {
-               newMessagesBox.state = "hidden"
-               chatLogView.scrollToBottom(true)
-           }
+            enabled: newMessagesBox.visible
+            cursorShape: enabled ? Qt.PointingHandCursor : undefined
```

Comment:
Wouldn't the enabled determine if the cursor will be shown? if that's the case, the cursorShape value can be simplified like this:
```
enabled: newMessagesBox.visible
cursorShape: Qt.PointingHandCursor
```

Reply:
No it doesn't . That's why I changed it manually.
Setting `enabled: false` just makes it that it doesn't accept clicks anymore